workflow/executor/emissary/emissary.go

```diff
@@ -129,7 +129,7 @@ func (e emissary) Wait(ctx context.Context, containerNames []string) error {
 			if e.isComplete(containerNames) {
 				return nil
 			}
-			time.Sleep(3 * time.Second)
+			time.Sleep(time.Second)
```

Comment:
making these loops righter has helped reduce e2e test time and make emissary and PS as fast as Docker